### PR TITLE
feat(i18n-UK): update Ukrainian translations

### PR DESCRIPTION
**Pull request by Entraft**
feat: Fixed some typos and added missing translations.

Signed-off-by: Entraft pasinc23@gmail.com

### CODE REVIEW

File: src/locales/uk.yaml [hunk 1]
```diff
@@ -74,92 +78,141 @@ app:
...
+      root_disabled: Root недоступний. Перевірте файли логів.
+      url:
+        klipper_config: https://www.klipper3d.org/Config_Reference.html#%{hash}
+        moonraker_config: https://moonraker.readthedocs.io/en/latest/configuration/#%{hash}
+        moonraker_telegram_bot_config: >-
+          https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config#%{hash}
+    url:
+      klipper_config: https://www.klipper3d.org/Config_Reference.html
+      moonraker_telegram_bot_config: https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config
+      moonraker_config: https://moonraker.readthedocs.io/en/latest/configuration/
```
Comment (pedrolamas) [inline src/locales/uk.yaml:120]: This bit doesn't look correct, there should not be any `url` under `tooltip` as it currently shows

**Review by pedrolamas — CHANGES_REQUESTED** [review]

File: src/locales/uk.yaml (hunk 1 above)
Comment (Entraft) [reply src/locales/uk.yaml:120]: I added them because in other languages this is done in such way (through a plugin for VS code i18n Ally)

Comment (Entraft) [reply src/locales/uk.yaml]: Thank you! Done.

File: src/locales/uk.yaml [hunk 2]
```diff
@@ -74,92 +78,136 @@ app:
...
       rename_dir: Перейменувати папку
       rename_file: Перейменувати файл
       upload_file: Завантаження файлу | Завантаження файлів
     tooltip:
       low_on_space: Мало місця на диску
+      root_disabled: Root недоступний. Перевірте файли логів.
+    url:
+      klipper_config: https://www.klipper3d.org/Config_Reference.html
+      moonraker_telegram_bot_config: https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config
+      moonraker_config: https://moonraker.readthedocs.io/en/latest/configuration/
```
Comment (pedrolamas) [inline src/locales/uk.yaml:115]: These are the used by Code Lens for deep linking, the hash params and the rest of the format needs to be kept as is (only thing to change would be the destination language, but I believe none of these components actually have ukranian support)

```suggestion
      klipper_config: 'https://www.klipper3d.org/Config_Reference.html#%{hash}'
      moonraker_config: 'https://moonraker.readthedocs.io/en/latest/configuration/#%{hash}'
      moonraker_telegram_bot_config: 'https://github.com/nlef/moonraker-telegram-bot/wiki/Sample-config#%{hash}'
```

**Review by pedrolamas — APPROVED** [review]
This is perfect, thank you @Entraft

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**